applications/openshift/api-server/api_server_api_priority_v1beta2_flowschema_catch_all/rule.yml

```diff
@@ -27,7 +27,7 @@ identifiers:
   cce@ocp4: CCE-86390-2
 
 platforms:
-  - ocp4.11 or ocp4.12 or ocp4.13
+  - ocp4.11 or ocp4.12 or ocp4.13 or ocp4.14 or ocp4.15 or ocp4.16
```

Comment:
It looks like https://github.com/ComplianceAsCode/content/blob/master/applications/openshift/api-server/api_server_api_priority_flowschema_catch_all/rule.yml is what's failing on 4.16 CI.